config-templates/config.php

```diff
@@ -463,6 +463,7 @@
      * In example when you are setting up a federation bridge.
      */
     'enable.saml20-idp' => false,
+    'enable.saml20-unsolicited' => true,
```

Comment:
I wonder if we should add another "enable." setting like this.
For starters, the current `enable.` flags seem a bit random already: you can enable or disable the idp. But not other aspects of SSP in a similar way. But ok, this is kind of historic.

When adding a functional change for SPs, I would add it to the SAML module's SP-related config (and docs), not to the global config. As that is where other behavioural things around SP's are configured.

Reply:
Not entirely true.. The adfs-idp is also enabled in a similar fashion.
I think I have addressed all your comments though